pkg/api/v1alpha1/cluster_webhook_test.go

```diff
@@ -27,7 +27,7 @@ func TestClusterDefault(t *testing.T) {
 }
 
 func TestClusterValidateUpdateManagementValueImmutable(t *testing.T) {
-	cOld := &v1alpha1.Cluster{}
+	cOld := createCluster()
```

Comment:
I know you didn't write this, but could we rename this to just `cluster()` or `baseCluster()`. I find that name misleading since it's not creating anything :)